### PR TITLE
Update Remove Users from an Organization Topic

### LINKED ISSUE #941: Update Remove User from Organization
**Affected Document:**

https://docs.astronomer.io/astro/add-user#remove-users-from-an-organization

**Problem Type (Priority):**

- [x] Error (Critical): This document has existing information that's incorrect and prevents readers from completing the documented setup (Add the `error/critical` label to this issue)
- [ ] Misleading (Important): This document could benefit from additional context, guidance, or clarity (Add the `error/FYI` label to this issue)
- [ ] Typo (Low): Any issue related to spelling/ punctuation/ formatting (Add the `error/typo` label to this issue)

**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):**

We are releasing in product support for Remove User from Organization on 07/21. Below is a screen cap of the feature in stage.

1. User performing the removal must be an Organization Owner
2. There must be at least 1 Organization Owner (can't remove yourself without making another user Org Owner)
3. To remove, go to People tab > Edit > Remove Member

image

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #941 

@chomipi88 my apologies, I didn't realize this new functionality was going to be available so soon! I would have included it with my other updates. The preliminary draft of the topic is ready for your review. Thanks!

### CODE REVIEW

**Comment by chomipi88** [comment]
LGTM

File: astro/add-user.md [hunk 1]
```diff
@@ -49,13 +50,13 @@ For more information on user roles, see [Manage user permissions on Astro](user-
 
 3. Click **Edit** next to the user you want to remove.
 4. Click **Remove member**.
-5. Click **Yes, continue** to confirm the removal.
+5. Click **Yes, Continue** to confirm the removal.
 
 ## Remove users from an Organization
 
-To remove users from an Organization, [submit a support request](astro-support.md). In your support request, provide the following information:
+Remove a user from an Organization when their role changes, or they leave the Organization. When you remove a user from an Organization, they are automatically removed from all of the Workspaces they had access to within the Organization.
```
Comment (paolaperaza) [inline astro/add-user.md:57]:
```suggestion
When you remove a user from an Organization, they are automatically removed from all of the Workspaces they had access to within the Organization.
```

Let's remove dis, don't really need it

File: astro/add-user.md [hunk 2]
```diff
@@ -5,12 +5,13 @@ id: add-user
 description: Add or remove users in the Cloud UI.
```
Comment (paolaperaza) [inline astro/add-user.md:5]: @kwcanuck Can you update the user-permissions table in https://docs.astronomer.io/astro/user-permissions to add a `Remove user from organization` line on Organization Owner?

**Review by paolaperaza — APPROVED** [review]

File: astro/add-user.md (hunk 2 above)
Comment (kwcanuck) [reply astro/add-user.md:5]: @paolaperaza the requested change has been implemented.

**Review by jwitz — APPROVED** [review]
LGTM!

**Pull request merged**